build/fbcode_builder/fbcode_builder.py

```diff
@@ -71,7 +71,7 @@ def _read_project_github_hashes():
         for filename in files:
             path = os.path.join(dirname, filename)
             with open(path) as f:
-                m_proj = re.match('^' + base_dir + '(.*)-rev\.txt$', path)
+                m_proj = re.match('^' + base_dir + '(.*)-rev\\.txt$', path)
```

Comment:
Can you revert this particular file from this PR?
It may sound like a weird request; fbcode_builder.py is a different project internally that is coincidentally mapped here when we export to github.
It has very opionated python version requirements and navigating those on the internal side of importing this PR could be a surprising amount of work, and none of it is related to or has benefit for watchman.

Reply:
Not a weird request at all. I reverted that file to the version in master.